Node/src/TeamsChatConnector.ts

```diff
@@ -149,10 +154,18 @@ export class TeamsChatConnector extends builder.ChatConnector {
     this.allowedTenants = null;
   }
 
-  public onQuery(commandId: string, handler: ComposeExtensionQueryHandlerType): void {
+  public onQuery(commandId: string, handler: ComposeExtensionHandlerType): void {
     this.queryHandlers[commandId] = handler;
   }
 
+  public onQuerySettingurl(handler: ComposeExtensionHandlerType) {
+    this.querySettingUrlHandler = handler;
+  }
+
+  public onSetting(handler: ComposeExtensionHandlerType) {
```

Comment:
onSettingUpdate seems fine to me, technically it's not updated yet until it's verified.